deploy/test-environments/fleet_api/src/agents_enrolled.py

```diff
@@ -0,0 +1,81 @@
+"""
+Wait for agents to be enrolled for a given policies
+If the expected number of agents is not enrolled within the timeout, the test will fail
+"""
+import sys
+import time
+from api.agent_policy_api import get_agents
+import configuration_fleet as cnfg
+from state_file_manager import state_manager
+from loguru import logger
+
+TIMEOUT = 600
+
+
+def get_expected_agents_mapping() -> dict:
+    """
+    Returns:
+        map: The name of the policy and the number of agents expected to be enrolled
+    """
+
+    logger.info("Loading agent policies state file")
+    policies_map = {}
+    for policy in state_manager.get_policies():
+        policies_map[policy.agnt_policy_id] = policy.expected_agents
+    return policies_map
+
+
+def get_actual_agents_mapping() -> dict:
+    """
+    Returns:
+        map: The name of the policy and the number of agents enrolled
```

Comment:
nit: let's use python's terminology

```suggestion
        dict: The name of the policy and the number of agents enrolled
```